clang/include/clang/Basic/AttrDocs.td

```diff
@@ -1885,6 +1885,18 @@ function, and no effect otherwise.
   }];
 }
 
+def SYCLIntelNumSimdWorkItemsAttrDocs : Documentation {
+  let Category = DocCatFunction;
+  let Heading = "num_simd_work_items (IntelFPGA)";
+  let Content = [{
+Applies to a device function/lambda function. Indicates the number of work
+items that should be processed in parallel. Valid values are positive integers.
+If ``intelfpga::num_simd_work_items`` is applied to a function called from a
+device kernel, the attribute is ignored and it is not propagated to a kernel.
+  }];
+}
+
+
```

Comment:
Remove extra line if it isn't meaningful.

Reply:
Done